netbox/dcim/filtersets.py

```diff
@@ -1812,6 +1817,13 @@ def filter_by_cable_end_b(self, queryset, name, value):
         # Filter by termination id and cable_end type
         return self.filter_by_cable_end(queryset, name, value, CableEndChoices.SIDE_B)
 
+    def _unterminated(self, queryset, name, value):
+        if value:
+            terminated_ids = queryset.filter(terminations__cable_end=CableEndChoices.SIDE_A).filter(terminations__cable_end=CableEndChoices.SIDE_B).values("id")
+            return queryset.exclude(id__in=terminated_ids)
+        else:
+            return queryset.filter(terminations__cable_end=CableEndChoices.SIDE_A).filter(terminations__cable_end=CableEndChoices.SIDE_B)
```

Comment:
These lines exceed the [120 character cut-off](https://docs.netbox.dev/en/stable/development/style-guide/#maximum-line-length-e501) and should be wrapped for better readability.